src/index.js

```diff
@@ -123,15 +127,15 @@ router
   })
   .get("/public/latest", async ctx => {
     const messages = await post.latest();
-    ctx.body = await publicView({ messages });
+    ctx.body = await latestView({ messages });
```

Comment:
There wasn't any access to `i18n` in the main index file, so I couldn't grab the description for each view and just pass it into the generic `publicView()`.  Instead I made separate view functions for each specific view in `views/index.js` where I could grab the i18n strings.  Not too elegant to make so many new functions like this, but probably best to keep i18n confined in the view side.